include/z64actor.h

```diff
@@ -171,12 +171,12 @@ typedef struct Actor {
     /* 0x085 */ u8 floorBgId; // Bg ID of the floor polygon directly below the actor
     /* 0x086 */ s16 wallYaw; // Y rotation of the wall polygon the actor is touching
     /* 0x088 */ f32 floorHeight; // Y position of the floor polygon directly below the actor
-    /* 0x08C */ f32 yDistToWater; // Distance to the surface of active waterbox. Negative value means above water
+    /* 0x08C */ f32 depthInWater; // Directed Distance to the surface of active waterbox. Negative value means water is below.
```

Comment:
Is "Distance" supposed to be capitalized here?
```suggestion
    /* 0x08C */ f32 depthInWater; // Directed distance to the surface of active waterbox. Negative value means water is below.
```